romanian/nalba.md

```diff
@@ -68,3 +68,7 @@ Bineinteles ca o sa bea. Ginel stia ca asta inseamna sa fii mafiot la urma urmei
 
 [Beau tot](Beau/Beau_paharul_pana_la_fund.md)
 
+Subit se simtea usor si isi imagina ca zboara - buna [potiunea](potiune_fermecata/potiune.md) asta, gandi el.
+
+Parea ca devenise un [pitigoi lustragiu](pitigoi/pitigoi.md) pentru incisivii
+[crocobalaurului](padurea_violet/croco/croco.md).
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.